Packs/Hackuity/Integrations/Hackuity/Hackuity.py

```diff
@@ -0,0 +1,694 @@
+    cvss_max = none_or_apply(args.get("cvss_max"), float)
+    vuln_type = args.get("vuln_type")
+    trs_min = none_or_apply(args.get("trs_min"), int)
+    trs_max = none_or_apply(args.get("trs_max"), int)
+    aggfindings = client.fetch_aggfindings(
+        asset_name=asset_name,
+        asset_type=asset_type,
+        attribute=attribute,
+        cvss_min=cvss_min,
+        cvss_max=cvss_max,
+        hy_global_only=hy_global_only,
+        limit=limit,
+        trs_min=trs_min,
+        trs_max=trs_max,
+        vuln_type=vuln_type,
+    )
+    outputs = []
+    for aggfinding in aggfindings:
+        output = json_format(
+            aggfinding,
+            {
+                "attributes": ["Attributes"],
+                "cvss": ["Score", "CVSS"],
+                "description": ["Description"],
+                "id": ["ID"],
+                "seen_first": ["Seen", "First"],
+                "trs": ["Score", "TRS"],
+            },
+            {
+                "seen_first": format_date,
+            },
+        )
+        output["Findings"] = json_format(
+            aggfinding["by_state"],
+            {
+                "total": ["Total"],
+                "open": ["Open"],
+                "closed": ["Closed"],
+                "ignored": ["Ignored"],
+            },
+        )
+        output["VulnTypes"] = [
+            {
+                "ID": vuln_type,
+                "Name": client.get_vulnerability_name(vuln_type),
+            }
+            for vuln_type in aggfinding["vuln_types"]
+        ]
+        outputs.append(output)
+    return CommandResults(
+        outputs_prefix="Hackuity.Vulnerabilities",
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            f"{'VulnDB' if hy_global_only else 'Provider'} vulnerabilities",
+            outputs,
+            headers=[
+                "VulnTypes",
+                "Description",
+                "Attributes",
+                "Score",
+                "Findings",
+                "Seen",
+            ],
+        ),
+    )
+
+
+def hackuity_dashboard_widgets_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    widgets = client.fetch_dashboard_widgets()
+    outputs = [
+        json_format(
+            widget,
+            {
+                "id": ["ID"],
+                "params": ["Params"],
+                "type": ["Type"],
+            },
+        )
+        for widget in widgets
+    ]
+    return CommandResults(
+        outputs_prefix="Hackuity.Dashboard.Widgets",
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            "Dashboard widgets",
+            outputs,
+            headers=["ID", "Type", "Params"],
+        ),
+    )
+
+
+def hackuity_dashboard_data_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    widget_id = args["widget_id"]
+    outputs = client.fetch_dashboard_widget_data(widget_id=widget_id)
```

Comment:
in case there's no widget with the given id, it would return an 404 error, in that case, we'd want to return a readable output stating that and not fail the command.

Reply:
Good catch!

Changed to the following, tell me if it's not the best way to display the error
![screenshot-2022-01-26-18:29:05](https://user-images.githubusercontent.com/3994389/151215219-a9042117-98ed-4346-bd14-6c77cc6ff9e5.png)

Reply:
looks perfect